pyproject.toml

```diff
@@ -0,0 +1,129 @@
+[tool.poetry]
+name = "megaqc"
+version = "0.3.0"
+description = "Collect and visualise data across multiple MultiQC runs"
+authors = ["Phil Ewels"]
+license = "GPLv3"
+readme = "README.rst"
+keywords = [
+        "bioinformatics",
+        "biology",
+        "sequencing",
+        "NGS",
+        "next generation sequencing",
+        "quality control",
+]
+homepage = "https://megaqc.info/"
+repository = "https://github.com/ewels/MegaQC/"
+documentation = "https://megaqc.info/docs/index.html"
+classifiers=[
+        "Development Status :: 4 - Beta",
+        "Environment :: Console",
+        "Environment :: Web Environment",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+        "Natural Language :: English",
+        "Operating System :: MacOS :: MacOS X",
+        "Operating System :: POSIX",
+        "Operating System :: Unix",
+        "Programming Language :: Python",
+        "Programming Language :: JavaScript",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Bio-Informatics",
+        "Topic :: Scientific/Engineering :: Visualization",
+    ]
+ packages=[
+    { include = "megaqc" },
+]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+        argon2-cffi="20.1"
```

Comment:
Fix indentation here. You might want to use the "Format Document" command in vscode.